scale/recipe/models.py

```diff
@@ -116,6 +116,7 @@ def create_recipe(self, recipe_type, data, event, superseded_recipe=None, delta=
             recipe_file = RecipeFile()
             recipe_file.recipe_id = recipe.id
             recipe_file.scale_file_id = input_file_id
+            recipe_file.recipe_input = data.get_input_file_name(input_file_id)
```

Comment:
I see how you're using this now. It should be renamed since you're not actually returning a file name and made more robust.